lib/src/drm/models/KeyMessage.ts

```diff
@@ -0,0 +1,14 @@
+import { MEDIA_KEY_MESSAGE_TYPES } from '../common/const/MEDIA_KEY_MESSAGE_TYPES';
+
+/**
+ * Represents a DRM key message for license requests.
+ *
+ * @group DRM
+ */
+export interface KeyMessage {
```

Comment:
Use `type` instead of `interface`

Reply:
Yea I noticed this after submitting PR. I already corrected these. Converted to type all of them.